lms/static/scripts/frontend_apps/components/dashboard/test/OrganizationActivity-test.js

```diff
@@ -175,20 +175,23 @@ describe('OrganizationActivity', () => {
       h_userid: ['123', '456'],
       assignment_id: [],
       course_id: [],
+      public_id: [],
```

Comment:
@acelaya while this seems to pass coverage it doesn't test we pass public_id when it's present, should we parametrize one other test? Which one? How? :grin:

Reply:
I can write a test if you want

Reply:
There https://github.com/hypothesis/lms/pull/6491

Reply:
That's great, I'm going to just merge it, thanks.